content/articles/how-to-add-pure-css-wave-animations-to-your-website/index.md

```diff
@@ -0,0 +1,270 @@
+  top: -197px;
+```
+- Set the animation-timing-function property using the `cubic-bezier ()` function. The function defines a Cubic Bezier curve which is defined by four points. We then set the `animation-iteration-count` to infinite to make the animation run forever.
+
+```css
+animation: wave-one 8s cubic-bezier (0.37, 0.44, 0.64, 0.54) infinite;
+```
+
+>Failing to set the other animation properties allows the computer to use the properties' default values while rendering the animation.
+
+
+- Give the elements a transform property with a value of `translate3d.` The translate3d is used to define a 3D translation (takes a vector as values) to create a 3-dimensional effect. 
+
+The final set of CSS rules defined for both `<div>` tags is as below:
+```css
+
+.blue-child {
+  background: url(https://s3-us-west-2.amazonaws.com/s.cdpn.io/85486/waves.svg) repeat-x; 
+  position: absolute;
+  min-height: 197px;
+  position: absolute;
+  width: 6400px;
+  top: -197px;
+  animation: wave-one 8s cubic-bezier (0.37, 0.44, 0.64, 0.54) infinite;
+  transform: translate3d (0, 0, 0);
+}
+
+```
+- Select the second child element of the class blue-child using the nth-type pseudo-class.
+
+```css
+.blue-child:nth-of-type(2) {
+  top: -174px;
+  ```
+We set it to be just below the first child element. Go on to set the animation property.
+
+- We name the two animations, wave-one and crests. We set the animation wave-one with the animation-delay property. Set it to a negative value to allow it to start first.
+  
+```
+animation: wave-one 8s cubic-bezier (0.37, 0.44, 0.64, 0.54) -.124s
+```
+- We have a second animation: `crests,` which will take **8s** to complete one cycle, has an animation-timing function of ease. This means the animation will start slow, fast, and slowly end. It will have a negative value as its `animation-delay` property's value. It will run forever (infinite), then give the element an opacity value of **1** to distinguish it from the first element.
+
+
+```css
+.blue-child:nth-of-type(2) {
+  top: -174px;
+  animation: wave-one 8s cubic-bezier (0.37, 0.44, 0.64, 0.54) -.124s infinite, crests 8s ease -1.24s infinite;
+  opacity: 1;}
+
+```
+>Defining the animations' sequence is done using the `@keyframes rule`, and it allows us to define how our animations 'behave' at certain times. 
+ 
+In our case, we will first define a keyframe for the `wave-one` animation. We define two time stops: **0%** and **100%.**
+
+- At the start state, we will set the left margin to **0,** then a negative value of **1600** pixels at the stop state. This allows the elements to move from right to left.
+```css
+@keyframes wave-one {
+  0% {
+    margin-left: 0;
+  }
+  100% {
+    margin-left: -1600px;
+  }
+}
+```
+
+- We define a transform property with the value translate3d and set the y-value to go from a negative to a positive then back to a negative value to make the second child element move up and down, completing the building of a wavy animation.
```

Comment:
You have typos on this paragraph. i'e `translate3d`.

Reply:
that is not a typo it is a property value.